RadarSDK/RadarLog.h

```diff
@@ -0,0 +1,37 @@
+//
+//  RadarLog.h
+//  RadarSDK
+//
+//  Copyright © 2021 Radar Labs, Inc. All rights reserved.
+//
+
+#import "Radar.h"
+
+/**
+ Represents a debug log.
+ */
+@interface RadarLog : NSObject
+
+/**
+ The levels for debug logs.
+ */
+@property (assign, nonatomic, readonly) RadarLogLevel level;
+
+/**
+ The log message.
+ */
+@property (nonnull, copy, nonatomic, readonly) NSString *message;
+
+/**
+ The datetime when the log occurred on the device.
+ */
+@property (nonnull, copy, nonatomic, readonly) NSDate *createdAt;
+
+- (instancetype _Nullable)initWithMessage:(NSString *_Nullable)message
+                                    level:(RadarLogLevel)level;
+
+- (NSDictionary *_Nonnull)dictionaryValue;
+
++ (NSArray<NSDictionary *> *_Nullable)arrayForLogs:(NSArray<RadarLog *> *_Nullable)logs;
```

Comment:
Should this be on `RadarAPIClient` instead?

Reply:
right now the pattern for these toArray type methods is to have it on the models
e.g. https://github.com/radarlabs/radar-sdk-ios/blob/master/RadarSDK/RadarPlace.m#L156

Reply:
Ah, ok.